ai_ml_app.py

```diff
@@ -25,9 +33,19 @@ def is_pto():
         message = request.form.get('message')
         prediction_score = int(pto_classifier.is_this_a_pto(message))
         response = jsonify({"score" : prediction_score, "message" : message})
-
     return response
 
+@app.route("/metrics", methods=["GET"])
+def metrics():
+    # update Prometheus gauges with CPU and RAM usage
+    cpu_percent = psutil.cpu_percent()
+    ram_info = psutil.virtual_memory()
+    cpu_usage.set(cpu_percent)
+    ram_usage.set(ram_info.percent)
+
+    # generate Prometheus metrics in the appropriate format
+    return Response(generate_latest(), mimetype="text/plain")
+
 #---START OF SCRIPT
 if __name__ == '__main__':
-    app.run(host='0.0.0.0', port=6464)
+    app.run(debug=True , host='0.0.0.0', port=6464)
```

Comment:
Why has debug been turned on? You should not use it on production.

Reply:
ok , removed debug mode